### PR TITLE
Image editor mobile layout update

### PR DESCRIPTION
**Pull request by renahlee**
### Overview

This PR:

- Updates the layout for the web image editor
- Refactors the image code to be simpler
- Bumps up max size and compression factor

**Note:** this could continue to use improvement but will revisit at a later point

Desktop:

Screenshot 2023-05-11 at 1 11 24 AM

Mobile web:

Screenshot 2023-05-11 at 1 20 16 AM

### CODE REVIEW

**Comment by aliceisjustplaying** [comment]
who do i have to bribe to get this merged

File: src/state/models/media/image.ts [hunk 1]
```diff
@@ -203,18 +208,21 @@ export class ImageModel implements RNImage {
...
     }
 
-    const ratioMultiplier = this.ratioMultipliers[this.aspectRatio ?? '1:1']
+    const ratioMultiplier =
+      this.ratioMultipliers[this.attributes.aspectRatio ?? '1:1']
 
-    // TODO: Ollie - should support up to 2000 but smaller images that scale
-    // up need an updated compression factor calculation. Use 1000 for now.
-    const MAX_SIDE = 1000
+    const MAX_SIDE = 2000
```
[the remark this answers is not in the document]
Comment (pfrazee) [reply src/state/models/media/image.ts:225]: yep!

File: src/state/models/media/image.ts [hunk 2]
```diff
@@ -223,7 +231,7 @@ export class ImageModel implements RNImage {
         {resize: ratioMultiplier > 1 ? {width: MAX_SIDE} : {height: MAX_SIDE}},
       ],
       {
-        compress: 0.7, // TODO: revisit compression calculation
+        compress: 0.9,
```
[the remark this answers is not in the document]
Comment (pfrazee) [reply src/state/models/media/image.ts:234]: Yep!

File: src/view/com/modals/EditImage.tsx [hunk 3]
```diff
@@ -18,148 +18,114 @@ import {Slider} from '@miblanchard/react-native-slider'
...
-          hint: 'Sets image aspect ratio to tall',
-          Icon: MaterialIcons,
-          name: 'do-not-disturb-alt',
-        },
-      } as const),
-    [],
-  )
-
-  type AspectRatio = keyof typeof ratios
+  const [altText, setAltText] = useState('')
```
[the remark this answers is not in the document]
Comment (pfrazee) [reply src/view/com/modals/EditImage.tsx:65]: Perfect

File: src/view/com/modals/EditImage.tsx [hunk 4]
```diff
@@ -184,25 +150,12 @@ export const Component = observer(function ({image, gallery}: Props) {
             ...(position !== undefined ? {position} : {}),
           }
         : {}),
-      ...manipulationAttributes,
```
[the remark this answers is not in the document]
Comment (pfrazee) [reply src/view/com/modals/EditImage.tsx:187]: Nice!

File: src/view/com/modals/EditImage.tsx [hunk 5]
```diff
@@ -345,30 +322,15 @@ export const Component = observer(function ({image, gallery}: Props) {
 const styles = StyleSheet.create({
   container: {
     gap: 18,
-    paddingVertical: 18,
-    paddingHorizontal: 12,
+    paddingHorizontal: isDesktopWeb ? undefined : 16,
```
[the remark this answers is not in the document]
Comment (pfrazee) [reply src/view/com/modals/EditImage.tsx:321]: I am _so_ sorry we lost your screenshots when it went public, after I didnt get on the review for this. I had no idea that would happen.

I'll intuit what you were highlighting and double check it.

**Review by pfrazee — APPROVED** [review]
Nice work on this! I really like how you solved the UI layout questions and the code is much cleaner.

**Pull request merged**